test/e2e/e2e_test.go

```diff
@@ -126,4 +123,39 @@ var _ = ginkgo.Describe("Chat", func() {
 			))
 		})
 	})
+
+	ginkgo.Context("update chat endpoint", func() {
+		ginkgo.It("should return valid response for updating group chat", func() {
+			createChatRequest := &chathttp.CreateChatDto{
+				Name: "Test Group Chat",
+				Type: uint8(chatdomain.GroupChatType),
+			}
+			createdChat := util.CreateChat(client, chatURL, util.AdminToken, createChatRequest)
+
+			updateChatRequest := &chathttp.UpdateChatDto{
+				Name: "Updated Group Chat",
+			}
+
+			expectedChatResponse := chathttp.ChatDto{
+				ID:        createdChat.ID,
+				Name:      "Updated Group Chat",
+				Type:      uint8(chatdomain.GroupChatType),
+				CreatedBy: util.AdminID,
+				UserChats: []chathttp.UserChatDto{
+					{
+						UserID: util.AdminID,
+						ChatID: createdChat.ID,
+					},
+				},
+			}
+
+			updatedChat := util.UpdateChat(client, chatURL, util.AdminToken, createdChat.ID, updateChatRequest, http.StatusOK)
+			gomega.Expect(updatedChat).To(gomega.BeComparableTo(
+				&expectedChatResponse,
+				cmpopts.IgnoreFields(chathttp.ChatDto{}, "CreatedAt", "UpdatedAt"),
+			))
+			gomega.Expect(updatedChat.UpdatedAt).ToNot(gomega.BeZero())
+			gomega.Expect(updatedChat.UpdatedAt.After(updatedChat.CreatedAt) || updatedChat.UpdatedAt.Equal(updatedChat.CreatedAt)).To(gomega.BeTrue(), "UpdatedAt should be after or equal to CreatedAt")
```

Comment:
```suggestion
			gomega.Expect(updatedChat.UpdatedAt.After(createdChat.CreatedAt)).To(gomega.BeTrue(), "updatedChat.UpdatedAt should be after createdChat.CreatedAt")
```
I think this is enough.